dangerfile.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import {fail, warn, danger} from 'danger';
+import fs from 'fs';
+
+// Takes a list of file paths, and converts it into clickable links
+const linkableFiles = (paths: Array<string>): string => {
+  const repoURL = danger.github.pr.head.repo.html_url;
+  const ref = danger.github.pr.head.ref;
+  const links = paths.map(path => {
+    return createLink(`${repoURL}/blob/${ref}/${path}`, path);
+  });
+  return toSentence(links);
+};
+
+// ["1", "2", "3"] to "1, 2 and 3"
+const toSentence = (array: Array<string>) : string => 
+  array.slice(0, array.length - 1).join(', ') + ' and ' + array.pop();
+
+// ("/href/thing", "name") to "<a href="/href/thing">name</a>"
+const createLink = (href: string, text: string): string => 
+  `<a href='${href}'>${text}</a>`;
+
```

Comment:
some of these ^ are functions from Danger RB, in the future I should probably move them into DangerJS too - the use case seems pretty universal